assets/style/style.css

```diff
@@ -11,24 +11,16 @@
   width: 100%;
   height: 100%;
 }
-.hero-background.is-transparent {
-  opacity: 0.3;
-}
 
 .page-header {
 text-align: center;
 
 
 }
 
-.hero-background.page-header{
-  color:gold;
-  text-align: center; 
-  margin-top: 20px; 
-  font-size: xx-large; 
-}
+
 .card {
-  background-color: rgba(0, 0, 0, 0.8);
-  color: rgb(226, 223, 223);
+  background-color: rgba(80, 68, 23, 0.481);
```

Comment:
The UI seems a little busy with the background logo and the song list with this opacity - what do you think of a 0.8 value for opacity here?